src/h3/ffi.rs

```diff
@@ -84,6 +84,31 @@ pub extern fn quiche_h3_conn_new_with_transport(
     }
 }
 
+#[no_mangle]
+pub extern fn quiche_h3_for_each_setting(
+    conn: &h3::Connection,
+    cb: extern fn(identifier: u64, value: u64, argp: *mut c_void) -> c_int,
+    argp: *mut c_void,
+) -> c_int {
+    let rc = match conn.peer_settings_raw() {
```

Comment:
Don't really need to assign the value to a variable and then return the variable, can just leave the `match` without the final `;` to have the value returned automatically.